trymito.io/pages/data-app.tsx

```diff
@@ -0,0 +1,136 @@
+import Footer from '../components/Footer/Footer';
+import Header from '../components/Header/Header';
+import pageStyles from '../styles/Page.module.css';
+import textImageSplitStyles from '../styles/TextImageSplit.module.css'
+import titleStyles from '../styles/Title.module.css';
+import homeStyles from '../styles/Home.module.css';
+
+// Import Icons & Background Grid
+
+import { classNames } from '../utils/classNames';
+import StreamlitAppGallery, { DATA_VERIFICATION_STREAMLIT_APP_LINK } from '../components/StreamlitAppGallery/StreamlitAppGallery';
+import CTAButtons from '../components/CTAButtons/CTAButtons';
+import DashboardCTACard from '../components/CTACards/DashboardCTACard';
+
+const DataApp: NextPage = () => {
+
+  return (
+    <>
+      <Head>
+        <title>Python Spreadsheet in Streamlit App | Mito </title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="description" content="Mito is a streamlit spreadsheet component that enables no-code spreadsheet automation with Pandas code generation from within a streamlit app. | Mito" />
+      </Head>
+      
+      <Header/>
+    
+      <div className={pageStyles.container}>
+
+        <main className={pageStyles.main}>
+
+            <section className={classNames(titleStyles.title_card)}>
+                <h1 className={titleStyles.title}>
+                    Upgrade Python scripts to interactive Streamlit Apps
+                </h1>
+                <p className={titleStyles.description}>
+                    Turn one hour of automation savings into tens of hours by sharing automation scripts through Streamlit dashboards.
+                </p>
+                <div className={homeStyles.cta_button_and_video_spacer}>
+                    <CTAButtons variant='download' align='center' displayProCTA={false}/>
+                </div>
+                    
+                <div id='video'>
+                    <video className={homeStyles.video} autoPlay loop disablePictureInPicture playsInline webkit-playsinline="true" muted>
+                        <source src={'/data-app/data-verification-app.mp4'} />
+                    </video>
+                </div>
+                <a href={DATA_VERIFICATION_STREAMLIT_APP_LINK} target="_blank" rel="noreferrer" className={pageStyles.link_with_p_tag_margins}>
+                    Try this app now →
+                </a>
+            </section>
+            <section className={textImageSplitStyles.text_image_section_container}>
+                <div className={pageStyles.subsection}>
+                    <div className={textImageSplitStyles.functionality_text}>
+                        <h1>
+                            From automation script, to <span className='text-highlight'>automation app</span>
+                        </h1>
+                        <p>
+                            It only takes one data analyst to build a script for an entire team to automate their similar workflows. 
```

Comment:
I think we should not talk about data analysts, but rather our user profile no?

Reply:
ooo yeah good catch lol